input/pagecontent/usecase-enrollment.md

```diff
@@ -1,84 +1,170 @@
+> In the background, the MSC checks the answers. If Patrick wouldn't be eligible for telemonitoring at the MSC, the request would be rejected here.
 
-Caroline cliks on "Complete request" and the request is sent to Zorg bij jou including all the information provided. 
+#### Contact information (PII)
 
-<b>Monitoring</b>
+The pop-up window now displays questions for basic contact information on patient Patrick and cardiologist Caroline (e.g. name, telephone number and email address). This information has been auto-populated from the EHR. Caroline just has to check and submit.
 
-As all information is provided, Zorg bij jou automatically accepts the tasks and provides a direct task status update, so Caroline can see that the task is received and subsequently accepted. 
+> In the background, the MSC checks the answers for completeness. The MSC automatically accepts the request and sends the patient an invite for a monitoring app.
+
+#### Request accepted
+
+The pop-up windows now responds that the MSC has accepted the request and has invited the patient to download a monitoring app. The telemonitoring request has been successfully assigned.
+
+#### Monitoring
 
 Caroline monitors the task in the EHR and sees that a few days later the status of the Task is updated to “in progress”, meaning that Patrick has downloaded and started to use the monitoring app.
 
 
-## Hospital X: Find existing CarePlan for Patient
+### Transactions
+
+#### Hospital X: Find existing CarePlan for Patient
 
 ```
-curl --request GET '{% raw %}{{cps-base-url}}{% endraw %}/CarePlan/?category=http://snomed.info/sct|135411000146103' \
---header 'Content-Type: application/json' \
---header 'Authorization: {% raw %}{{access-token}}{% endraw %}'
+curl --request GET '{% raw %}{{cps-base-url}}{% endraw %}/CarePlan/?category=http://snomed.info/sct|135411000146103' --header 'Content-Type: application/json' \
 ```
 
-## Hospital X: Create Task 
+#### Hospital X: Create Task 
 
 assumption no CarePlan was found (so no Task.basedOn), create a Bundle with a new Task and a copy of the referred ServiceRequest and Condition:
 
 ```
-curl --request POST '{% raw %}{{cps-base-url}}{% endraw %}/' \
---header 'Content-Type: application/json' \
---header 'Authorization: {% raw %}{{access-token}}{% endraw %}' \
---data '{% include Bundle-cps-bundle1.json %}'
+curl --request POST '{% raw %}{{cps-base-url}}{% endraw %}/'  --header 'Content-Type: application/json' --header 'Authorization: {% raw %}{{access-token}}{% endraw %}' \
+--data '{% include Bundle-cps-bundle-01.json %}'
+```
+
+#### CarePlanService: Create subscriptions, careplan, careteam and notifications
```

Comment:
The requests below are quite unreadable from within the IG. Maybe also add a link to the json files so the body becomes more readable without the need to copy/paste/clean up & format?

Reply:
ok, replaced codeblacoks by links